nameserver/mocks_test.go

```diff
@@ -75,6 +75,10 @@ func (mz *mockedZoneWithRecords) AddRecord(ident string, name string, ip net.IP)
 	notImplWarn()
 	return nil
 }
+func (mz *mockedZoneWithRecords) DeleteRecords(ident *string, ip *net.IP, name *string) int {
```

Comment:
I agree; using the initial value (i.e. what you'd get from `var x string` and `var x net.IP`) as a wildcard would be better, I think.